topic-operator/src/main/java/io/strimzi/operator/topic/OperatorAssignedKafkaImpl.java

```diff
@@ -262,11 +262,7 @@ private void executeReassignment(File reassignmentJsonFile, String zookeeper, Lo
                     || line.contains("There is an existing assignment running")
                     || line.contains("Failed to reassign partitions")) {
                 throw new TransientOperatorException("Reassigment failed: " + line);
-            } else if (line.contains("Successfully started reassignment of partitions.")) {
-                return true;
-            } else {
-                return null;
-            }
+            } else return line.contains("Successfully started reassignment of partitions.");
```

Comment:
This looks to me like it would return `true` or `false` and not `true` or `null`. Do you know for sure that this does not matter? Also, can we keep the same style and use `{` and `}` here?

Reply:
Yeah. Spotbugs is complaining about returning null when expecting boolean. I ashet @tombentley for doublechecking this logic. Otherwise, we can just add an exclude there.